### ISSUE TITLE
Kibana yaml is empty

### ISSUE THREAD

**Issue by spartan782** [rocknsm/rock#360]
The `/etc/kibana/kibana.yml` file in 2.3 is deleted and replaced with only 3 options. We should just make the changes inline and leave the rest of the yaml documented incase someone wants to go poking around or make their own customizations.

**Comment by dcode** [comment]
We can probably make a jinja2 filter or  ansible module that can round trip using https://pypi.org/project/ruamel.yaml/0.6/

**Comment by dcode** [comment]
I've found that several projects have been using the `yedit` ansible module. I think we can adopt this to really simplify a lot of our app configurations. 

[yedit role repo](https://github.com/kwoodson/ansible-role-yedit)

**Comment by jonneyj** [comment]
Agreed If you would like some help making these changes let me know I would not mind helping out.

**Comment by neu5ron** [comment]
can we add the reference fill separately? 
such as `/etc/kibana/kibana.yml` being the one with limited options then have `/etc/kibana/kibana.reference.yml`

this is how the beats modules do it.

**Comment by jonneyj** [comment]
A reference file is a good idea. From what I'm seeing by default all of the config settings in Kibana are commented out so all you would have to do is have a template file and then at a block of setting to the top of it when you toss it on during the install.  If it's an ansible related task or can be I would be happy to help if needed.

**Comment by bndabbs** [comment]
Reference files are what I had in mind when I made the change to only put the configured settings in the file. The reason we haven't completely moved in that direction is that it complicates the use of tools like @dcode was suggesting.
